city_scrapers/spiders/il_regional_transit.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+import scrapy
+
+import re
+from datetime import datetime
+
+from city_scrapers.spider import Spider
+
+
+# The RTA's Board and other meetings are are displayed on their
+# website via an iframe from a different domain.
+class RegionaltransitSpider(Spider):
+    name = 'il_regional_transit'
+    agency_id = 'Regional Transportation Authority '
+    long_name = 'Regional Transportation Authority'
+    timezone = 'America/Chicago'
+
+    allowed_domains = ['www.rtachicago.org', 'rtachicago.granicus.com']
+    start_urls = ['http://www.rtachicago.org/about-us/board-meetings']
+    domain_root = 'http://www.rtachicago.org'
+
+    def parse_iframe(self, response):
+        for item in response.css('.committee'):
+            start = self._parse_start(item)
+            name = self._parse_name(item)
+            data = {
+                '_type': 'event',
+                'name': name,
+                'event_description': response.meta['event_description'],
+                'start': start,
+                'end': {'date': None, 'time': None, 'note': ''},
+                'all_day': False,
+                'timezone': 'America/Chicago',
+                'location': self._parse_location(),
+                'documents': self._parse_documents(item),
+                'sources': [{'url': response.url, 'note': ''}],
+            }
+            data['id'] = self._generate_id(data)
+            data['status'] = self._generate_status(data, '')
+            data['classification'] = self._parse_classification(data)
+            yield data
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` a dict that follows the `Open Civic Data
+        event standard <http://docs.opencivicdata.org/en/latest/data/event.html>`_.
+        """
+        url = response.css('iframe::attr(src)').extract_first()
+        desc_xpath = '//*[text()[contains(.,"The RTA Board")]]/text()'
+        description = response.xpath(desc_xpath).extract_first()
+        meta = {
+            'event_description': description,
+            # Disable built-in RobotsTxt middleware for this request.
+            'dont_obey_robotstxt': True,
+        }
+        yield scrapy.Request(url, callback=self.parse_iframe, meta=meta)
+
+    @staticmethod
+    def _parse_classification(item):
+        name = item.get('name', '')
+        if 'Committee' in name:
+            return 'Committee'
+        if 'Board' in name:
+            return 'Board'
+        if 'Commission' in name:
+            return 'Commission'
```

Comment:
Hey @mkrump , could you add a case here for "citizens advisory"? It looks like they don't have any meetings posted right now, but it's called a variety of things: "citizens advisory council", "citizens advisory board", "citizens advisory committee". In any case, we would want to return "Citizens Advisory Council" to differentiate these meetings from one run by RTA board members. Otherwise, looks good! Thanks for knocking out all these spiders!